provider/go.mod

```diff
@@ -31,6 +31,7 @@ require (
 )
 
 replace (
+	github.com/containerd/containerd => github.com/containerd/containerd v1.4.11 // Work around https://github.com/advisories/GHSA-c2h3-6mxw-7mvq
```

Comment:
Just curious if/when we can upgrade this 1.5.7+?

Reply:
I'm not sure how to figure out which module is pulling in this dependency, so I just stuck with the same minor version. I noticed the same thing in pulumi/pulumi, so we may need to update a common dependency.

FYI @pulumi/platform

Reply:
```
anton@Antons-MacBook-Pro:~/pulumi-kubernetes/provider> go mod why github.com/containerd/containerd/sys 
go mod why github.com/containerd/containerd/sys 
# github.com/containerd/containerd/sys
github.com/pulumi/pulumi-kubernetes/provider/v3/pkg/provider
helm.sh/helm/v3/pkg/action
helm.sh/helm/v3/internal/experimental/registry
github.com/deislabs/oras/pkg/content
github.com/containerd/containerd/content/local
github.com/containerd/containerd/sys
```

Reply:
Ah, my google-fu was letting me down yesterday. Thanks!